dist/index.js

```diff
@@ -51,6 +52,13 @@ var _H = 'H'.charCodeAt(0);
 var _Y = 'Y'.charCodeAt(0);
 var _S = 's'.charCodeAt(0);
 
+function changeDpiBuffer(format, blob, dpi) {
```

Comment:
what is the blob argument here? a blob or a buffer?

Reply:
sorry, the variable name is confusing, it should be buffer

Reply:
So a couple of suggestion
1. rename it buffer

2. With this addition the users have to specify the format by themselves. That is fine, but they have to know how to  do it. Docs will be a way for sure, but also adding to the exports the current 2 supported formats is an help, as you did for the new function. so that they can just reference the 2 formtted strings `image/jpeg` and `image/png`.